### PR TITLE
Add missing tags to XTREME

### PR DESCRIPTION
**Pull request by mariosasko**
Add missing tags to the XTREME benchmark for better discoverability.

### CODE REVIEW

File: datasets/xtreme/README.md [hunk 1]
```diff
@@ -1,8 +1,462 @@
...
+  tatoeba.tur:
+  - tr
+  tatoeba.urd:
+  - ur
+  tatoeba.vie:
+  - vi
+  SQuAD:
+  - en
+licenses:
+- apache-2.0
```
Comment (lhoestq) [inline datasets/xtreme/README.md:428]: Not sure about the license, SQuAD says cc-by-4.0 for example

**Review by lhoestq — APPROVED** [review]
Awesome thanks !

File: datasets/xtreme/README.md (hunk 1 above)
Comment (mariosasko) [reply datasets/xtreme/README.md:428]: Fixed!

File: datasets/xtreme/README.md [hunk 2]
```diff
@@ -1,8 +1,467 @@
...
+- cc-by-4.0
+- cc-by-2.0
+- cc-by-sa-4.0
+- other-Licence Universal Dependencies v2.5
+- other-Attribution-NonCommercial 4.0 International
```
Comment (lhoestq) [inline datasets/xtreme/README.md:433]: I believe this is `cc-by-nc-4.0` (Creative Commons Attribution Non Commercial 4.0 International) ?
```suggestion
- cc-by-nc-4.0
```

Comment (mariosasko) [reply datasets/xtreme/README.md:433]: This is the license of the [XNLI](https://paperswithcode.com/dataset/xnli) dataset, and yes, you are right.

Comment (lhoestq) [reply datasets/xtreme/README.md]: Ok cool ! Let's merge now :D

**Pull request merged**